resources/metadata/integrationArtifactUpdateConfiguration.yaml

```diff
@@ -0,0 +1,95 @@
+metadata:
+  name: integrationArtifactUpdateConfiguration
+  description: Update integration flow Configuration parameter
+  longDescription: |
+    With this step you can update the value for a configuration parameters of a designtime integration flow using the OData API. Learn more about the SAP Cloud Integration remote API for configuration update of the integration flow parameter [here](https://help.sap.com/viewer/368c481cd6954bdfa5d0435479fd4eaf/Cloud/en-US/83733a65c0214aa6acba035e8640bb5a.html).
```

Comment:
Just FYI: You can put this in multiple rows. When the md file is rendered it will automatically be put in the same row for you. Unless there is an empty line between two rows, then a new paragraph is created.